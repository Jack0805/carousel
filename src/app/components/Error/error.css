.error-container {
    display: flex;
    width: 83%;
    justify-content: flex-start;
}