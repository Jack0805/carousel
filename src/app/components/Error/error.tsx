'use client'
import React from 'react';
import './error.css'

interface ErrorProps {
    errorMessage: string
}
const Error = ({errorMessage}: ErrorProps) => {
    return (
        <div className="error-container">
            <p className='bold-header'>{errorMessage}</p>
        </div>
    );
};

export default Error;