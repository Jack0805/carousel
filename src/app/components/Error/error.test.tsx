import { render } from '@testing-library/react';
import React from 'react';
import Error from './error'
import '@testing-library/jest-dom';
describe('Error Component', () => {

    it('should render correctly', async () => {
        const { container, getByText } = render(
                <Error errorMessage={'Something is wrong !!'}/>
        );

        const textElement = getByText('Something is wrong !!');

        expect(textElement).toBeInTheDocument();

        expect(container.firstChild).toMatchSnapshot();
    });
})
