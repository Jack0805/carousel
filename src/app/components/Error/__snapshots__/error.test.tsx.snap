// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Component should render correctly 1`] = `
<div
  class="error-container"
>
  <p
    class="bold-header"
  >
    Something is wrong !!
  </p>
</div>
`;
